label_studio_converter/brush.py

```diff
@@ -35,6 +35,7 @@
 from PIL import Image
 from collections import defaultdict
 from itertools import groupby
+import pycocotools.mask
```

Comment:
Need to add pycocotools as a new requirement

Reply:
updated

Reply:
Seems that pycocotools should be built for Windows users.
It will break simple installation for non-engineers users who is not aware of this process.
Can we exclude this requirement and use something else?

Reply:
I would use this approach: https://stackoverflow.com/a/43090648/1045399

and try something like this:
```
try:
  import pycocotools
  pycocotools_enabled = True
except ImportError:
  pycocotools_enabled = False
  print('To export brush to COCO you should install pycocotools: pip install label-studio[pycocotools]')
```
... 
then you can use pycocotools_enabled to control further converter behaviour

Reply:
I tried to install pycocotools on Windows and found that it required [Visual C++ build tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/).

I'm not familiar with Windows. Is VC++ build tools considered required for LabelStudio installation?

Reply:
VC++ build tools is not required. 
Let's make it like Max suggested: add try block in imports and check if pycocotools is available in COCO convert

Reply:
@cdpath do you have any news?

Reply:
@cdpath do you have any news?